cmd/iam.go

```diff
@@ -858,17 +858,11 @@ func (sys *IAMSys) GetUserInfo(ctx context.Context, name string) (u madmin.UserI
 }
 
 // GetUserPolicies - get policies attached to a user.
-func (sys *IAMSys) GetUserPolicies(ctx context.Context, name string) (p []string, err error) {
+func (sys *IAMSys) GetUserPolicies(name string) (p []string, err error) {
 	if !sys.Initialized() {
 		return p, errServerNotInitialized
 	}
 
-	select {
-	case <-sys.configLoaded:
-	default:
-		sys.store.LoadUser(ctx, name)
-	}
-
```

Comment:
so the user doesn't have to be loaded from disk yet to fetch policies? or is it that we don't care to return or block the call for this?

Reply:
This is intended to work like `PolicyDBGet`, without getting policies of associated groups. I mistakenly copied the LoadUser part, but it shouldn't be necessary to fetch the policies since `PolicyDBGet` doesn't require it. It also seems to work fine without those lines.

Reply:
It will work, the code exists to block and load in situations when the server has not loaded the user yet from disk. It is not trivial to reproduce such a scenario - so wanted to know the intention.

Reply:
Makes sense. I was just checking how policyDBGet worked, and since it doesn't load the user I assumed this shouldn't have to either.